exercises/jigsaw-puzzle/instructions.md

```diff
@@ -0,0 +1,21 @@
+### Task 1
+
+Provide a function `jigsawData` that receives a part of the information and fills in the blanks. If the provided information is insufficient to calculate the rest, there should be an error "Insufficient information". The full information about the jigsaw puzzle contains
+
+- `pieces`: Number of pieces
+- `border`: Number of border pieces
+- `inside`: Number of inside pieces
+- `rows`: Number of rows of pieces
+- `columns`: Number of columns of pieces
+- `aspecRatio`: Aspect ratio of columns / rows
+- `format`: Format of the puzzle: `"portrait" / "square" / "landscape" / "3d ball"`
+
+For the rows and columns of a 3d ball puzzle, assume a square distribution.
```

Comment:
What is a 3d ball and what does a square distribution mean?

Reply:
If it's 3D, wouldn't there be three dimensions? A square is a 2D object. Would this be a cube shape?

Reply:
I removed the 3d configuration